src/modules/person/domain/personenkontext.service.ts

```diff
@@ -35,4 +35,13 @@ export class PersonenkontextService {
 
         return { ok: true, value: personenkontexte };
     }
+
+    public async findById(id: string): Promise<Result<PersonenkontextDo<true>, DomainError>> {
```

Comment:
maybe rename the method to findPersonKontextById to match the method names of the other services? see organisation and person services.

Reply:
Done.